LDAR_Sim/src/file_processing/output_processing/summary_outputs.py

```diff
@@ -99,4 +99,63 @@ def generate_emissions_summary(directory: str, outputs_mapper: SummaryOutputMapp
         ),
         file_processing_const.Multi_Sim_Output_Const.EMIS_PATTERN,
     )
-    return emissions_summary_df
+    estimated_df: pd.DataFrame = generate_emissions_estimation_summary(directory, outputs_mapper)
+    merged = pd.merge(
+        emissions_summary_df,
+        estimated_df,
+        on=[
+            output_file_constants.SummaryFileColumns.CommonColumns.PROGRAM_NAME,
+            output_file_constants.SummaryFileColumns.CommonColumns.SIMULATION_NUMBER,
+        ],
+        how="outer",
+    )
+    # Fill NaN values with zeros
+    merged.fillna(0, inplace=True)
+    return merged
+
+
+def generate_emissions_estimation_summary(directory: str, outputs_mapper: SummaryOutputMapper):
```

Comment:
Design Question: What is the point of making these just to basically drop all the columns?

Reply:
Which columns in particular is this comment referring to?